AustrianPatient/AustrianPatient.xml

```diff
@@ -192,9 +194,6 @@
       </type>
     </element>
     <element id="Patient.animal">
-      <extension url="http://hl7.org/fhir/StructureDefinition/structuredefinition-explicit-type-name">
-        <valueString value="Animal" />
-      </extension>

```

Comment:
Is this modification on purpose?
I'm asking because it has nothing to do with the patientReligion.
Could it be that a new Forge version caused changes in the way the data is saved?

Reply:
This was not on purpose. I tried to find out, why this has been removed, with no success. My personal opinion is that it was removed by forge on saving the profile. I double-checked in the other branches as well, and the extension element is present there, therefore, I will add it again.